sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/Analyzer.scala

```diff
@@ -720,6 +722,14 @@ class Analyzer(
     }
   }
 
+  case class ResolveNamespace(catalogManager: CatalogManager)
+    extends Rule[LogicalPlan] with LookupCatalog {
+    def apply(plan: LogicalPlan): LogicalPlan = plan resolveOperators {
+      case UnresolvedNamespace(CatalogAndNamespace(catalog, ns)) =>
+        ResolvedNamespace(catalog.asNamespaceCatalog, ns)
+    }
```

Comment:
Does it mean the `ParsedStatement` from parser will turn to use `UnresolvedNamespace`? Currently, the catalogs in statements are resolved at `ResolveCatalogs`. Will we need to refactor `ResolveCatalogs` due to this change?

Reply:
Yes, see
https://github.com/apache/spark/pull/26847#issuecomment-567516386 
> We can have an extra rule to catch commands with v1 relation, and convert them to v1 commands. This can help us get rid of the duplicated code between ResolveCatalogs and ResolveSessionCatalog